orangecontrib/network/tests/test-arcslist.net

```diff
@@ -0,0 +1,12 @@
+*Vertices 6

```

Comment:
How do I convince Tox to copy this file?! Test fail because it's apparently missing.

Reply:
add this to the PACKAGE_DATA in setup.py

```'orangecontrib.network.tests': ['*']```

Reply:
Thanks!